ethexe/consensus/src/validator/initial.rs

```diff
@@ -33,13 +42,19 @@ use gprimitives::H256;
 #[display("INITIAL in {:?}", self.state)]
 pub struct Initial {
     ctx: ValidatorContext,
-    state: State,
+    state: WaitingFor,
 }
 
-#[derive(Debug, PartialEq, Eq)]
-enum State {
-    WaitingForChainHead,
-    WaitingForSyncedBlock(SimpleBlockData),
+#[derive(Debug)]
+enum WaitingFor {
```

Comment:
It'd be great to have scheme of state transitioning for each participant in consensus (a bit later)